sql/core/src/test/scala/org/apache/spark/sql/streaming/TransformWithStateTTLTest.scala

```diff
@@ -143,43 +143,54 @@ abstract class TransformWithStateTTLTest
         AddData(inputStream, InputEvent("k1", "put", 1)),
         // advance clock to trigger processing
         AdvanceManualClock(1 * 1000),
+        // In the primary index, we should have that k1 -> [1].
```

Comment:
nit: (1, 61000), but please feel free to leave it as it is if you intend to do this to cover all types

Reply:
Actually, for all types, we do store the expiration with it. This is just a typo. I fixed this everywhere.